manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -407,6 +416,9 @@ global:
   # Currently, two options are supported: "third-party-jwt" and "first-party-jwt".
   jwtPolicy: "third-party-jwt"
 
+  # This flag can be set true in order to be able to run the sidecar in gateway mode.
+  sidecarInHybridMode: false
```

Comment:
are we using this flag still?

Reply:
@kfaseela , I'll remove this. You're right, this flag is not used anymore.